stdlib/public/TensorFlow/CMakeLists.txt

```diff
@@ -21,16 +21,14 @@ endif()
 find_package(TensorFlow REQUIRED)
 message(STATUS "Building TensorFlow.")
 
-include_directories(BEFORE "${TF_INCLUDE_DIR}")
-
 set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
 set(swift_stdlib_compile_flags "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}")
 list(APPEND swift_stdlib_compile_flags "-Xllvm" "-sil-inline-generics")
 list(APPEND swift_stdlib_compile_flags "-Xllvm" "-sil-partial-specialization")
 list(APPEND swift_stdlib_compile_flags "-Xfrontend" "-enable-sil-ownership")
 list(APPEND swift_stdlib_compile_flags "-swift-version" "4")
 list(APPEND swift_stdlib_compile_flags "-force-single-frontend-invocation")
-list(APPEND swift_stdlib_compile_flags "-Xcc" "-I${TF_INCLUDE_DIR}")
+# list(APPEND swift_stdlib_compile_flags "-Xcc" "-I${TF_INCLUDE_DIR}")
```

Comment:
is this change intentional?

Reply:
I believe this is superceeded by the modulemap, and thus is not needed.